frontend/src/assets/lang/app/en.json

```diff
@@ -285,7 +285,14 @@
         },
         "edit": "Edit",
         "cancel": "Cancel",
-        "save": "Save",
+        "save": {
+            "save": "Save",
+            "error": {
+                "title": "Invalid Safe Operation",
+                "detail": "Trying to save while no item is selected"

```

Comment:
To get the same as the Dutch one

Maybey: "U are trying to save an item without selecting it."